test/common/config/grpc_mux_impl_test.cc

```diff
@@ -139,6 +139,27 @@ TEST_F(GrpcMuxImplTest, MultipleTypeUrlStreams) {
   expectSendMessage("foo", {}, "");
 }
 
+// Validate behavior when dynamic context parameters are updated.
```

Comment:
Maybe this test already does this, but can we verify that after clearing a force send node we go back to not sending it? We have had issues with this type of thing in the past. If the test already tests this maybe add a comment?